.github/workflows/call.deploy-subgraph.yml

```diff
@@ -35,12 +35,14 @@ jobs:
 
     steps:
       - name: Show contexts
+        env:
+          HEAD_REF: ${{ github.head_ref }}
         run: |
           echo github.event_name: ${{ github.event_name }}
           echo github.sha: ${{ github.sha }}
           echo github.repository: ${{ github.repository }}
           echo github.ref: ${{ github.ref }}
-          echo github.head_ref: ${{ github.head_ref }}
+          echo github.head_ref: "$HEAD_REF"
```

Comment:
where is this suggestion coming from:

echo github.head_ref: "$HEAD_REF"

Reply:
and the suggestion: https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-an-intermediate-environment-variable

Reply:
what about `github.ref`

Reply:
hmm, actionlint doesn't complain about it, but it is in that page, will add ref and any other things which might be an issue